README.md

```diff
@@ -45,6 +45,7 @@ main directory of this repository.
 | `--message-starts-with`   | `-l`  | (**OR**) Filter for certain commit messages.                              |
 | `--output`                | `-o`  | An output file for the commits per day in CSV format.                     |
 | `--quiet`                 | `-q`  | Hide the output of commit messages.                                       |
+| `--verbose`               | `-V`  | Always show the entire output.                                            |
```

Comment:
The new option is named `-V` in its abbreviated form.  Please note the majuscule.

Some projects use rather `-v` (minuscule) therefore, including Git, others apply this convention in order to reserve the minuscule for the version request (which is about to be introduced by another Pull Request in the near future).

Do you prefer the Git way, also regarding the verbosity levels? :wrench:

Reply:
Hmm, I mean is the version number important enough for a shorthand flag? I think a `-v` for verbose might be more fitting. At least, as long as `-v` does not mean anything else.

Reply:
:wrench: :heavy_check_mark: